test/DynamoCoreWpfTests/PublishPackageViewModelTests.cs

```diff
@@ -99,7 +100,34 @@ public void CanPublishLateInitializedJsonCustomNode()
         }
 
 
-        [Test, Category("Failure")]
+        [Test]
+        public void NewPackageDoesNotThrow_NativeBinaryIsAddedAsAdditionalFile_NotBinary()
```

Comment:
Does the dependencies field in the pkg.json being empty indicate that the native assembly is not added as a dependent binary but an additional file?

Reply:
the dependencies field points to other packages